benchmark/opperf/opperf.py

```diff
@@ -99,15 +99,15 @@ def _parse_mxnet_context(ctx):
     # CLI Parser
 
     # 1. GET USER INPUTS
-    parser = argparse.ArgumentParser(
-        description='Run all the MXNet operators (NDArray) benchmarks')
+    parser = argparse.ArgumentParser(description='Run all the MXNet operators (NDArray) benchmarks')
```

Comment:
Update the description here since you can also support types other than NDArray operators now.

Reply:
Done.